src/compass/s1_geocode_metadata.py

```diff
@@ -122,6 +126,131 @@ def run(cfg, burst, fetch_from_scratch=False):
         f"{module_name} burst successfully ran in {dt} (hr:min:sec)")
 
 
+def geocode_calibration_luts(geo_burst_h5, burst, cfg, 
+                             dec_factor=40):
+    '''
+    Geocode the radiometric calibratio paremeters,
+    and write them into output HDF5.
+
+    Parameters
+    ----------
+    geo_burst_h5: h5py.files.File
+        HDF5 object as the output product
+    burst: s1reader.Sentinel1BurstSlc
+        Sentinel-1 burst SLC
+    cfg: GeoRunConfig
+        GeoRunConfig object with user runconfig options
+    dec_factor: int
+        Decimation factor to downsample the slant range pixels for LUT
+    '''
+    dem_raster = isce3.io.Raster(cfg.dem)
+    epsg = dem_raster.get_epsg()
+    proj = isce3.core.make_projection(epsg)
+    ellipsoid = proj.ellipsoid
+    burst_id = str(burst.burst_id)
+    geo_grid = cfg.geogrids[burst_id]
+    radar_grid = burst.as_isce3_radargrid()
+
+    date_str = burst.sensing_start.strftime("%Y%m%d")
+    burst_id_date_key = (burst_id, date_str)
+    out_paths = cfg.output_paths[burst_id_date_key]
+
+    # Common initializations
+    threshold = cfg.geo2rdr_params.threshold
+    iters = cfg.geo2rdr_params.numiter
+    scratch_path = out_paths.scratch_directory
+
+    root_path = '/science/SENTINEL1'
+
+    # Designate radiometric calibration parameter to geocode
+    calibration_dict = {
+        'gamma':burst.burst_calibration.gamma,
+        'sigma_naught':burst.burst_calibration.sigma_naught,
+    }
+
+    # define the geogrid for calbration LUT
+    radargrid_calibration = radar_grid.multilook(dec_factor,
+                                                 dec_factor)
+    geogrid_calibration = isce3.product.GeoGridParameters(
```

Comment:
```suggestion
    calibration_radargrid = radar_grid.multilook(dec_factor,
                                                 dec_factor)
    calibration_geogrid = isce3.product.GeoGridParameters(
```
nit - rename with adjective then noun makes more sense to me

`calibration` to `cal` still makes sense
```suggestion
    cal_radargrid = radar_grid.multilook(dec_factor,
                                                 dec_factor)
    cal_geogrid = isce3.product.GeoGridParameters(
```